torch/csrc/jit/passes/onnx/onnx_log.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <torch/csrc/Export.h>
+#include <ostream>
+#include <string>
+
+namespace torch {
+namespace jit {
+
+TORCH_API bool is_onnx_log_enabled();
+
+TORCH_API void set_onnx_log_enabled(bool enabled);
+
+TORCH_API void set_onnx_log_output_stream(std::ostream* out_stream);
+
+TORCH_API std::ostream& _get_onnx_log_output_stream();
+
+#define ONNX_LOG(...)                            \
+  if (is_onnx_log_enabled()) {                   \
```

Comment:
Defines are not guaranteed to be invoked in the same namespace, so please specify namespace fully (same as you do on line 19) 
```suggestion
  if (::torch::jit::is_onnx_log_enabled()) {                   \
```